### PR TITLE
Accept SVG import as a stable feature

### PR DESCRIPTION
**Pull request by t-paul**
Changes needed for #2702.

### CODE REVIEW

File: src/import_svg.cc [hunk 1]
```diff
@@ -5,39 +5,65 @@
...
 			PRINTD("SVG shape");
 			const auto page = dynamic_cast<libsvg::svgpage *>(shape_ptr.get());
+			double unit_scale = 1.0;
 			if (page) {
-				width = page->get_width();
-				height = page->get_height();
+				const libsvg::length_t h = page->get_height();
+				switch (h.unit) {
+				case libsvg::unit_t::UNDEFINED:
+				case libsvg::unit_t::NONE:
```
Comment (kintel) [inline src/import_svg.cc]: NONE doesn't look like it's used any longer

Comment (t-paul) [reply src/import_svg.cc]: It's used inside `parse_length()` -> `get_unit()` (in libsvg/util.cc) when parsing numbers that have no unit.

**Comment by t-paul** [comment]
Yep, that will be a little bit trickier than usual as the current tests do auto-scale, so they can't pick up simple scale changes. But we certainly need to extend the unit handling and add test cases...

**Comment by kintel** [comment]
We can always render a manually sized object next to it for comparison

**Comment by t-paul** [comment]
Right, and having the test case include 2 different units so relative changes are picked up too.

**Pull request merged**